docs/performance/reduce-reconciliation.md

```diff
@@ -0,0 +1,138 @@
+In all likelihood, the primary source of performance gains for your app will come from reducing the amount of work that Roact's reconcilation process requires. This is accomplished by:
```

Comment:
This page needs a title!

Reply:
Technically, the docs will automatically give it the title of the page itself if one isn't present.  But I'll add it for consistency's sake.